src/ProtocolGateway.Core/Instrumentation/SafePerformanceCounter.cs

```diff
@@ -4,16 +4,28 @@
 namespace Microsoft.Azure.Devices.ProtocolGateway.Instrumentation
 {
     using System;
+#if !NETSTANDARD1_3
     using System.Diagnostics;
+#endif
 
-    public sealed class SafePerformanceCounter
+    public sealed class SafePerformanceCounter : IPerformanceCounter
```

Comment:
are we using SafePerformanceCounter in net standard (considering counter impl is no-op)? If not, there's no point if-ing PerformanceCounter usage and instead "if out" the whole class for net standard version.